evap/evaluation/models.py

```diff
@@ -916,9 +937,22 @@ def send_to_user(self, user, courses=None, cc=True):
         else:
             cc_addresses = []
 
+        grade_documents = len(grade_document_courses) > 0
+        evaluation_results = len(evaluation_results_courses) > 0
+
         mail = EmailMessage(
-            subject = self.render_string(self.subject, {'user': user, 'courses': courses}),
-            body = self.render_string(self.body, {'user': user, 'courses': courses}),
+            subject = self.render_string(self.subject, {
+                'grade_documents': grade_documents,
+                'evaluation_results': evaluation_results
+            }),
+            body = self.render_string(self.body, {
+                'user': user,
+                'courses': courses,
+                'grade_documents': grade_documents,
+                'evaluation_results': evaluation_results,
+                'grade_document_courses': grade_document_courses,
+                'evaluation_results_courses': evaluation_results_courses
```

Comment:
maybe we should provide example email templates in the initial data. what do you think? right now it is completely unclear what they should look like